GeoInterfaceMakie/src/GeoInterfaceMakie.jl

```diff
@@ -0,0 +1,101 @@
+module GeoInterfaceMakie
+
+using GeoInterface
+import MakieCore as MC
+import MakieCore
+import GeometryBasics as GB
+import GeoInterface as GI
+import Makie
+
+
+function _plottype(geom)
+    plottype_from_geomtrait(GI.geomtrait(geom))
+end
+function plottype_from_geomtrait(::Union{GI.LineStringTrait, GI.MultiLineStringTrait})
+    MC.Lines
+end
+function plottype_from_geomtrait(::Union{GI.PointTrait, GI.MultiPointTrait})
+    MC.Scatter
+end
+function plottype_from_geomtrait(::Union{GI.PolygonTrait,GI.MultiPolygonTrait, GI.LinearRingTrait})
+    Makie.Poly
+end
+function pttype(geom)
+    if GI.is3d(geom)
+        GB.Point3f
+    else
+        GB.Point2f
+    end
+end
+function points(geom)::Union{Vector{GB.Point2f}, Vector{GB.Point3f}}
```

Comment:
TODO: 
https://github.com/JuliaGeo/GeoInterface.jl/issues/70#issuecomment-1272649198

Reply:
@rafaqz is this what you had in mind?

Reply:
Sorry I should have been more specific. 

You can call `GeoInterface.getpoint(obj)` and get an iterator over all the points in nested geometries. This has optimizations for e.g. shapefile, where it will just give you the vectors of points that are already iterators.